config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  get 'authors/index'
+
   root "books#index"
 
   resources :books, only: [:index, :show]
+  resources :authors
```

Comment:
What do you think about limiting this to the actions we're implementing here, so as not to have unused routes?